src/components/ContentHighlights/CurrentContentHighlightItemsHeader.jsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
-import { ActionRow } from '@edx/paragon';
-import { useParams } from 'react-router-dom';
+import { ActionRow, Skeleton } from '@edx/paragon';
+import PropTypes from 'prop-types';
 import ContentHighlightHelmet from './ContentHighlightHelmet';
 import DeleteHighlightSet from './DeleteHighlightSet';
 
-const CurrentContentHighlightItemsHeader = () => {
-  const { highlightSetUUID } = useParams();
-
-  const highlightTitle = highlightSetUUID;
-
-  const titleName = `${highlightTitle} - Highlights`;
-
+const CurrentContentHighlightItemsHeader = ({ isLoading, highlightTitle }) => {
+  if (isLoading) {
+    return (
+      <div data-testid="header-skeleton">
+        <Skeleton />
```

Comment:
nit: perhaps something along the following instead?

```jsx
<ActionRow>
  <h2><Skeleton /></h2> // rendering `Skeleton` as child of `h2` means the `Skeleton` will inherit the `h2` styles (e.g., its height).
  <ActionRow.Spacer />
  <Skeleton />
</ActionRow>
```

such that the skeleton state more closely matches the styles that will replace the skeletons (i.e., an `h2` and a CTA button to the right).